developing_a_metrics_plan.adoc

```diff
@@ -144,6 +144,14 @@ Because no two open source communities are the same, every community will natura
 
 Communities should always establish metrics collaboratively and agree on them colectively. Hearing from a diverse group of community members is important for ensuring the metrics are inclusive and not just focusing on the work of a subset of the community. For example, if internationalization is important for your project, you want to ensure that your metrics reflect internationalization work. It's also a good practice to review the metrics periodically with the community and discuss if any adjustments are needed for your metrics. Even within the same community, you will likely need to evolve your metrics along with the community as your needs change.
 
+Here are a few of the more common metrics open source communities may use to measure results:
```

Comment:
Rather than a bullet point, can we list them in a paragraph? A bullet point makes it look like these are must have items. Whenever, people ask me what set of metrics should I have in the community, I turn around and ask "what are your goals?"

How about something like this work? 

"Depending on the community, you may see metrics for things like backlog, contributor diversity (e.g. organizational, geographical, etc.), first-time contributors, popular discussion topics, review efficiency (e.g. for bugs/code), etc."

Reply:
Love this. Thanks. I think it accomplishes that task well. I incorporated it. See what you think.

Reply:
Looks great. Thanks @semioticrobotic!